apps/dashboard/src/main.tsx

```diff
@@ -106,7 +108,7 @@ const router = createBrowserRouter([
             children: [
               {
                 path: ROUTES.WELCOME,
-                element: <WelcomePage />,
+                element: IS_SELF_HOSTED ? <Navigate to={ROUTES.ROOT} replace /> : <WelcomePage />,
```

Comment:
Is there a reason of why not keeping it? I think it might be helpful for onboarding, even on OSS. Wdyt?

Reply:
I think we need to revisit and restructure this page, some elements only partially make sense in the current context. For example, having both “Create Account” and “Invite Member” feels a bit out of place.
I’m not against having them, but we haven’t had a proper discussion around this yet.
A cleaner approach could be to focus on more actionable, relevant items like: "Create Workflow" "Add Inbox to Your App"
![image](https://github.com/user-attachments/assets/131d9a43-c755-4b85-b3e9-916889c119a4)

Let me know what you think. Open to feedback or other suggestions on how to streamline this page.

Reply:
i brought it back without the non-relevant steps, and the get started on the footer is back as well.
![image](https://github.com/user-attachments/assets/7014b475-7e99-4320-834a-b3b17e5793b6)